docs/UserGuide.md

```diff
@@ -28,7 +28,7 @@ Assistant who can type fast, TAilor can get your contact management tasks done f
 
    * **`list`** : Lists all contacts.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com m/CS2103T g/W12` : Adds a student
+   * **`add`**`n/John Doe a/A0123456P e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact
```

Comment:
Change the usecase to include the module and group too.

Reply:
oh I'm surprised this got resolved this way while merging my master and alt branch! Oversight on my end and didn't even hinder the test cases hmm. Getting onto solving this asap- thanks Neel!